napari/utils/tree/group.py

```diff
@@ -30,14 +31,24 @@ class Group(NestableEventedList[NodeType], Node):
         A name/id for this group, by default "Group"
     """
 
-    def __init__(self, children: Iterable[NodeType] = (), name: str = "Group"):
-        NestableEventedList.__init__(self, children, basetype=Node)
+    def __init__(
+        self,
+        children: Iterable[NodeType] = (),
+        name: str = "Group",
+        basetype=Node,
+    ):
         Node.__init__(self, name=name)
+        NestableEventedList.__init__(self, data=children, basetype=basetype)
+        self._selection: Selection[NodeType] = Selection()
```

Comment:
We could probably add a setter which took the set and then instead of creating a new evented set just replace the existing one with its current contents

Reply:
added a setter.  it doesn't create a new `Selection` object, it just updates it in two steps to match the provided data